runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -242,6 +246,12 @@ public static CorfuRuntimeParametersBuilder builder() {
          */
         private MetricRegistry metricRegistry;
 
+        private boolean microMeterRegistryEnabled;
+
+        private Duration microMeterLoggingInterval;
+
+        private String configuredLogger;
```

Comment:
can we extract these 3 variables into a MetricsConfig class?

Reply:
done